build.gradle

```diff
@@ -82,38 +93,32 @@ jar {
                     "Plugin-Build-Timestamp": new Date().format("yyyy-MM-dd'T'HH:mm:ssZ"),
                     "Git-Commit"            : "${-> project.ext.gitProps['git.commit.id.abbrev']}",
                     "Git-Branch"            : "${-> project.ext.gitProps['git.branch']}",
-                    "Published-By"          : nexusUsername]) //Pass via commandline properties -PnexusUsername which work for below as well. 
+                    "Published-By"          : "nexusUsername"])
```

Comment:
@RxL-Nikhil-Kanyal Use project.findProperty("nexusUsername") ?: System.getenv("NEXUS_USERNAME") in place of "nexusUsername"

Reply:
done